.github/workflows/build_and_release.yml

```
@@ -0,0 +1,59 @@
+name: Build and Release
+
+on:
+  workflow_dispatch: # Allows manual triggering
+    inputs:
+      tag:
+        description: 'Tag name'     
```

yamllint

[error] trailing spaces

(trailing-spaces)